apps/web/pages/success.tsx

```diff
@@ -355,6 +368,12 @@ export default function Success(props: SuccessProps) {
                         is24h={is24h}
                       />
                     </div>
+                    {reschedule && !!rescheduleReason?.length && (
+                      <>
+                        <div className="font-medium">{t("reschedule_reason_success_page")}</div>
+                        <div className="col-span-2 mb-6 last:mb-0">{rescheduleReason}</div>
+                      </>
+                    )}
```

Comment:
can you move this above "what"? @gitstart

Reply:
sure thing